src/geo.c

```diff
@@ -210,33 +210,22 @@ void addReplyDoubleDistance(client *c, double d) {
 }
 
 /* Helper function for geoGetPointsInRange(): given a sorted set score
- * representing a point, and a GeoShape, appends this entry as a geoPoint
- * into the specified geoArray only if the point is within the search area.
+ * representing a point, and a GeoShape, checks if the point is within the search area.
  *
- * returns C_OK if the point is included, or C_ERR if it is outside. */
-int geoAppendIfWithinShape(geoArray *ga, GeoShape *shape, double score, sds member) {
-    double distance = 0, xy[2];
-
+ * returns C_OK if the point is within search area, or C_ERR if it is outside. */
+int geoWithinShape(GeoShape *shape, double score, double *xy, double *distance) {
```

Comment:
please document the inputs (and outputs)

Reply:
@oranagra documented it. Can you check if it's OK now.